lib/widgets/detail_overview_card_widget.dart

```diff
@@ -0,0 +1,82 @@
+import 'package:flutter/material.dart';
+import 'package:tech_byte/models/product_model.dart';
+import 'package:tech_byte/utils/colors.dart';
+import 'package:tech_byte/utils/constants.dart';
+import 'package:tech_byte/widgets/rating_widget.dart';
+
+class TBDetailOverviewCard extends StatelessWidget {
+  final TBProduct product;
```

Comment:
Do not pass the whole object, because sometimes the model is much bigger than what should be displayed. Add param by param.